app/controllers/activity_executions_controller.rb

```diff
@@ -13,7 +13,11 @@ def index
 
     respond_to do |format|
       format.html
-      format.csv { send_exported_data(@activity_executions) }
+      if params[:export] == 'units'
```

Comment:
ich würde das eben in den UnitActivityExecutionsController verschieben, dann muss man die Unterscheidung hier nicht machen. => Löst auch die beiden Linting-Errors

Reply:
moved code to UnitActivityExecutionsController, reverted changes in the ActivityExecutionController